src/main/java/uk/ac/cam/cl/dtg/segue/api/Constants.java

```diff
@@ -410,6 +410,7 @@ public PGInterval getPGInterval() {
     public static final String[] ADDRESS_FIELDNAMES = {"addressLine1", "addressLine2", "town", "county", "postalCode"};
     public static final String SEARCHABLE_CONTENT_FIELDNAME = "searchableContent";
     public static final String VISIBLE_TO_STUDENTS_FIELDNAME = "visibleToStudents";
+    public static final String QUIZ_RUBRIC_FIELDNAME = "rubric";
```

Comment:
I'm not certain this is needed, please do correct me if I'm wrong though!

Reply:
Not sure either but I figured it was not doing any harm.